src/main/scala/LagTracker.scala

```diff
@@ -0,0 +1,36 @@
+package chess
+
+import LagTracker._
+
+case class LagTracker(
+    quota: Centis = Centis(200),
+    history: Option[DecayingStats] = None
```

Comment:
I don't love this architecture.  every player gets a LagTracker object which has two states, initial and with data, rather than having an empty LagTracker that converts to a real one.